vpr/src/route/router_lookahead_map_utils.cpp

```diff
@@ -5,7 +5,12 @@
 #include "vtr_math.h"
 #include "route_common.h"
 
-util::PQ_Entry::PQ_Entry(
+namespace util {
+
+/* Number of CLBs I think the average conn. goes. */
+static const int CLB_DIST = 3;
```

Comment:
I believe this can be deleted, as it is unused

Reply:
Removed.